zbx_redis_stats.py

```diff
@@ -120,13 +120,14 @@ def default():
 
 	a = []
 	for i in server_info:
-		a.append(Metric(args.hostname, ('redis[%s]' % i), server_info[i]))
+		a.append(Metric(hostname, ('redis[%s]' % i), server_info[i]))
```

Comment:
I think here the `hostname` should be the same as  the `hostname` configured as in Zabbix Web UI (a.k.a., the control panel). If get it from argument, then make sure the argument is the same as the configured `hostname` in Web UI.